package.json

```diff
@@ -41,7 +41,7 @@
     "bottleneck": "1.5.x",
     "event-stream": "3.2.2",
     "express": "4.11.2",
-    "hiredis": "0.4.0",
+    "hiredis": "~0.4.1",
```

Comment:
This should not include the tilde. We've had issues where DNSChain failed to work on future installs because of an updated library, so we want to stick to precise version numbers.